src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java

```diff
@@ -17,7 +20,7 @@
 	private String m_keyspace;
 
 	@Inject
-	public CassandraClientImpl(CassandraConfiguration config)
+	public CassandraClientImpl(CassandraConfiguration config, Tracer tracer)
```

Comment:
You can use the GlobalTracer holder below instead of injecting here.  If not yet initialized, it will Noop, and then start tracing as soon as initialized.